src/database/factories/user.factory.ts

```diff
@@ -2,18 +2,21 @@ import * as Faker from 'faker';
 import { define } from 'typeorm-seeding';
 import { User } from '@entities/user.entity';
 import { genSaltSync, hashSync } from 'bcrypt';
+import { Genders } from '@constants/enums/genders';
 
 define(User, (faker: typeof Faker) => {
   const firstName = faker.name.firstName();
   const lastName = faker.name.lastName();
   const email = faker.internet.email(firstName, lastName);
   const password = 'password123';
+  const randomGender = faker.random.arrayElement(Object.values(Genders));
```

Comment:
```suggestion
  const gente = faker.random.arrayElement(Object.values(Genders));
```

To keep with the standard variable names.

Reply:
Great idea Emi ! One question, do you mean const gender, not gente , right?